docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -156,16 +167,18 @@ components:
             $ref: '#/components/schemas/StellarTransaction'
         external_transaction_id:
           type: string
+          description: "The id of an external transaction to be handled by the Anchor Backend."
         custodial_transaction_id:
           type: string
         source_account:
           type: string
-          description: The SEP-12 user ID of the source account.
+          description: "The Stellar account who sent the payment that originated this event. Should be present if this event was originated from a Stellar payment."
         destination_account:
           type: string
-          description: The SEP-12 user ID of the destination account.
+          description: "The Stellar account who received the payment that originated this event. Should be present if this event was originated from a Stellar payment."
         customers:
           type: object
+          description: "This field should contain the `id` or the **deprecated** `account+memo` identification pair of the sending and receiving customers created through [SEP-12](https://stellar.org/protocol/sep-12)."
```

Comment:
Why is the account+memo deprecated? These fields are sent in the SEP-12 requests which get forwarded to the anchor, so why are we deprecating them here?

Reply:
I updated the description in https://github.com/stellar/java-stellar-anchor-sdk/pull/412/commits/a08c05507a781745f8931e393cc6e972bb825972.